README.md

```diff
@@ -52,6 +52,7 @@ yarn db:reset
 ### Rest client
 
 - `GET /platforms`
+- `POST /platforms`
 - `GET /platforms/:slug`
 - `GET /games/`
 - `GET /games/:slug`
```

Comment:
```suggestion
- `GET /platforms`;
- `POST /platforms`;
- `GET /platforms/:slug`;
- `GET /games/`;
- `GET /games/:slug`;
```

Reply:
This is more HTTP listing than javascript.